feature/qos/otg_tests/wrr_traffic_test/wrr_traffic_test.go

```diff
@@ -983,6 +991,15 @@ func ConfigureQoS(t *testing.T, dut *ondatra.DUTDevice) {
 	q := d.GetOrCreateQos()
 	queues := netutil.CommonTrafficQueues(t, dut)
 
+	if dut.Vendor() == ondatra.NOKIA {
```

Comment:
Similar to this comment: https://github.com/openconfig/featureprofiles/pull/2024/files#r1326530972

Reply:
Added new deviation `qos_queue_requires_id`.